schematic/utils/schema_utils.py

```diff
@@ -17,6 +17,8 @@ def load_schema_into_networkx(schema):
         # adding nodes to the graph
         node = {}
         for (k, value) in record.items():
+            if isinstance(value,dict) and "@language" in value.keys():
```

Comment:
@GiaJordan could you add a short in-code comment on why this line is needed to load schema.org? Otherwise the PR looks good!

Reply:
@milen-sage sure thing!

Reply:
Perfect - thanks! 🚀